### ISSUE TITLE
[JENKINS-58519] Add OWASP to BlueOcean view

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/blueocean-plugin#4995]
<p><a href="https://issues.jenkins.io/browse/JENKINS-37437" title="Pipeline integration for OWASP checker" class="issue-link" data-issue-key="JENKINS-37437"><del>JENKINS-37437</del></a> added support for OWASP dependency checker that shows a neat UI report:</p>

<p><span class="image-wrap" style=""><img src="https://issues.jenkins.io/secure/attachment/48010/48010_Screenshot+2019-07-16+16.59.57.png" width="400" style="border: 0px solid black" /></span></p>

<p>It would be great to have this colorful bar visible in a blueocean view, below the branch info details and/or in the Tests tab. </p>

---
<details><summary><i>Originally reported by <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/warden.png" title="warden's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=warden">warden</a>, imported from: <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-58519" target="_blank">Add OWASP to BlueOcean view</a></i></summary>
<i><ul>
<li><b>status</b>: Closed
<li><b>priority</b>: Minor
<li><b>component(s)</b>: blueocean-plugin, dependency-check-jenkins-plugin
<li><b>resolution</b>: Won't Do
<li><b>resolved</b>: 2023-11-04T00:23:07+00:00
<li><b>votes</b>: 1
<li><b>watchers</b>: 4
<li><b>imported</b>: 2025-12-01
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p><a href="https://issues.jenkins.io/browse/JENKINS-37437" title="Pipeline integration for OWASP checker" class="issue-link" data-issue-key="JENKINS-37437"><del>JENKINS-37437</del></a> added support for OWASP dependency checker that shows a neat UI report:</p>

<p><span class="image-wrap" style=""><img src="https://issues.jenkins.io/secure/attachment/48010/48010_Screenshot+2019-07-16+16.59.57.png" width="400" style="border: 0px solid black" /></span></p>

<p>It would be great to have this colorful bar visible in a blueocean view, below the branch info details and/or in the Tests tab. </p></pre>
</details>
</details>
<details><summary><i>1 attachment</i></summary>

- [Screenshot 2019-07-16 16.59.57.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/48010/Screenshot%202019-07-16%2016.59.57.png)
  > ![Screenshot 2019-07-16 16.59.57.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-misc/refs/heads/main/attachments/48010/Screenshot%202019-07-16%2016.59.57.png)
</details>

**Comment by jenkins-infra-bot** [comment]
- _Remote links associated with this issue:_


  - [JEP-203](https://github.com/jenkinsci/jep/blob/master/jep/203/README.adoc)
  - [JEP-204](https://github.com/jenkinsci/jep/blob/master/jep/204/README.adoc)

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/sspringett.png" title="sspringett's avatar" /> sspringett:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-58519?focusedId=371340&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-371340">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>I believe providing a UI for blue ocean is still blocked by JEP-203 and JEP-204</p>



<p>See</p>
<ul>
	<li><a href="https://github.com/jenkinsci/jep/blob/master/jep/203/README.adoc" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/jep/blob/master/jep/203/README.adoc</a></li>
	<li><a href="https://github.com/jenkinsci/jep/blob/master/jep/203/README.adoc" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/jep/blob/master/jep/204/README.adoc</a></li>
</ul>


<p> </p>

<p> </p>
</pre>
</details>

</ul>
</details>

<p>I believe providing a UI for blue ocean is still blocked by JEP-203 and JEP-204</p>



<p>See</p>
<ul>
	<li><a href="https://github.com/jenkinsci/jep/blob/master/jep/203/README.adoc" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/jep/blob/master/jep/203/README.adoc</a></li>
	<li><a href="https://github.com/jenkinsci/jep/blob/master/jep/203/README.adoc" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/jep/blob/master/jep/204/README.adoc</a></li>
</ul>


<p> </p>

<p> </p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/oleg_nenashev.png" title="oleg_nenashev's avatar" /> oleg_nenashev:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-58519?focusedId=371607&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-371607">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>It is <img class="emoticon" src="https://issues.jenkins.io/images/icons/emoticons/sad.png" height="16" width="16" align="absmiddle" alt="" border="0"/></p>
</pre>
</details>

</ul>
</details>

<p>It is <img class="emoticon" src="https://issues.jenkins.io/images/icons/emoticons/sad.png" height="16" width="16" align="absmiddle" alt="" border="0"/></p>
